share/p2p/peers/manager.go

```diff
@@ -418,6 +428,31 @@ func (m *Manager) isBlacklistedPeer(peerID peer.ID) bool {
 	return !m.connGater.InterceptPeerDial(peerID)
 }
 
+func (m *Manager) arePeerProtocolsCompatible(peerID peer.ID) bool {
```

Comment:
I'd call it `isSupported()` or `supportsProtocols()`

Reply:
Cleaner, changed 👍🏻